packages/arb-validator/cmd/arb-stressed-validator/arb-stressed-validator.go

```diff
@@ -52,7 +54,7 @@ func main() {
 	}
 }
 
-func createStressedManager(rollupAddress common.Address, client arbbridge.ArbClient, contractFile string, dbPath string) (*rollupmanager.Manager, error) {
+func createStressedManager(ctx context.Context, rollupAddress common.Address, client arbbridge.ArbClient, contractFile string, dbPath string) (*rollupmanager.Manager, error) {
 	return rollupmanager.CreateManager(
 		context.Background(),
```

Comment:
This looks like it should be `ctx` instead

Reply:
good catch